app/course/courseApp.js

```diff
@@ -52,4 +52,5 @@ var SELECT_ALL_COURSE_ROWS = "SELECT_ALL_COURSE_ROWS";
 var DESELECT_ALL_COURSE_ROWS = "DESELECT_ALL_COURSE_ROWS";
 var OPEN_COURSE_DELETION_MODAL = "OPEN_COURSE_DELETION_MODAL";
 var CLOSE_COURSE_DELETION_MODAL = "CLOSE_COURSE_DELETION_MODAL";
-var DELETE_MULTIPLE_COURSES = "DELETE_MULTIPLE_COURSES";
\ No newline at end of file
+var DELETE_MULTIPLE_COURSES = "DELETE_MULTIPLE_COURSES";
```

Comment:
Your PR shouldn't involve touching this file, I suspect something went wrong git-ing, I'll look at it with you next time we're in the office.